docs/whats-new.md

```diff
@@ -78,7 +72,7 @@ Note: This change is mainly relevant to developers who write custom deck.gl laye
 
 ## Experimental Features
 
-As usual, deck.gl 4.2 contains a number of experimental features, e.g. multi viewport and first person viewport support. These features are still being finalized and the APIs have not been frozen, but can still be accessed by early adopters. See the roadmap article for more information on these.
+As usual, deck.gl 4.2 contains a number of experimental features, e.g. `multi viewport`, `first person viewport` and `viewport transitions`. These features are still being finalized and the APIs have not been frozen, but can still be accessed by early adopters. See the roadmap article for more information on these.
```

Comment:
Nit: Back ticks are usually for code symbols like actual class and function names, maybe normal quotes are more appropriate?